recipe/meta.yaml

```diff
@@ -25,13 +23,40 @@ requirements:
   host:
     - nasm
 
+# This is here just to fool the linter until
+#  https://github.com/conda-forge/conda-smithy/issues/833
+# is fixed
 test:
   commands:
-    - test -f ${PREFIX}/include/x264.h         # [unix]
-    - test -f ${PREFIX}/lib/libx264.a          # [unix]
-    - test -f ${PREFIX}/lib/libx264.dylib      # [osx]
-    - test -f ${PREFIX}/lib/libx264.so         # [linux]
-    - x264 --help                              # [unix]
+
+outputs:
+  - name: x264
+    build:
+      run_exports:
+         - {{ pin_subpackage('libx264') }}
+    files:
+      - bin/x264
+    test:
+      commands:
+        - x264 --help                                          # [unix]
+
+  - name: libx264
+    version: {{ lib_version }}
+    build:
+      run_exports:
+        - libx264 >={{ so_number }}.00000000,<{{ so_number|int + 1 }}.00000000
```

Comment:
shouldn't we use `release_date` here?

Reply:
the reason we don't is that release date has little to do with version compatibility

Reply:
You should use `{{ pin_subpackage('libx264') }}` here same as above. but I really prefer we don't do this and go ahead with #18 instead.